src/DesktopVirtualization/DesktopVirtualization.Autorest/README.md

```diff
@@ -47,21 +44,32 @@ In this directory, run AutoRest:
 > see https://aka.ms/autorest
 
 ``` yaml
-commit: 50175f111e9c899249e79eb082a75fb8a7aba0e2
+commit: 0feca76719343b0cb1e6a9d6064c7037827706ca
 require:
   - $(this-folder)/../../readme.azure.noprofile.md
 sanitize-names: true
 subject-prefix: 'Wvd'
 input-file:
-- $(repo)/specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/stable/2023-09-05/desktopvirtualization.json
+- $(repo)/specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/stable/2024-04-03/desktopvirtualization.json
 
-module-version: 2.1.0
+module-version: 6.0.0
 title: DesktopVirtualizationClient
 # For new modules, please avoid setting 3.x using the use-extension method and instead, use 4.x as the default option
-use-extension:
-  "@autorest/powershell": "3.x"
+# use-extension:
+#  "@autorest/powershell": "3.x"
+
+#v4 migration settings
+keep-pec-and-plr: true
+disable-transform-identity-type: true
+flatten-userassignedidentity: false
 
 directive:
+  - where:
+      variant: ^(Create|Update)(?!.*?(Expanded|JsonFilePath|JsonString))
```

Comment:
What is the purpose of this directive? I see new parameters `-JsonFilePath` and `-JsonString` are generated in the cmdlets and docs. Is it expected?

Reply:
It was intended to fix part of the autorest v3 to v4 bugs, but it may not be needed. Lets remove it and see if we still pass CI